platform/linux-generic/odp_classification.c

```diff
@@ -1782,3 +1782,95 @@ uint64_t odp_pmr_to_u64(odp_pmr_t hdl)
 {
 	return _odp_pri(hdl);
 }
+
+static
+void print_cos_ident(struct cos_s *cos)
+{
+	ODP_PRINT("%s(%d)\n", cos->name, (int)cos->index);
+}
+
+static
+void print_queue_ident(odp_queue_t q)
+{
+	odp_queue_info_t info;
+
+	if (odp_queue_info(q, &info))
+		ODP_PRINT("%" PRIx64, odp_queue_to_u64(q));
+	else
+		ODP_PRINT("%s", info.name);
+
+	ODP_PRINT("\n");
+}
+
+static
+void print_hex(const void *vp, int len)
+{
+	const uint8_t *p = vp;
+
+	for (int i = 0; i < len; i++)
+		ODP_PRINT("%02x", (int)*p++ & 0xff);
```

Comment:
Bitwise operations with signed values likely produce new static code checker warnings. Is cast to int needed? May be cast to uint32_t instead of int, and do bitwise operation with 0xffu.